source/server/services/login-service.js

```diff
@@ -0,0 +1,32 @@
+Space.Object.extend(Space.accounts, 'LoginService', {
```

Comment:
I have an object in the Job Queue package that's playing a similar role: Translating and publishing events. I've named it `Space.jobQueue.EventPublisher`. `LoginService` seems to infer that it's playing an active role in the login -> we ask this object to login, whereas when you read the code it's just broadcasting events and is reactive. `Space.accounts.LoginEventPublisher' is maybe a more suited name?